shared/contracts/BaseTemplate.sol

```diff
@@ -157,6 +157,28 @@ contract BaseTemplate is APMNamehash, IsContract {
         _acl.createPermission(_grantee, _tokenManager, _tokenManager.REVOKE_VESTINGS_ROLE(), _manager);
     }
 
+    function _mintTokens(ACL _acl, TokenManager _tokenManager, address[] _holders, uint256[] _stakes) internal {
+        _createPermissionForTemplate(_acl, _tokenManager, _tokenManager.MINT_ROLE());
+        for (uint256 i = 0; i < _holders.length; i++) {
+            _tokenManager.mint(_holders[i], _stakes[i]);
+        }
+        _removePermissionFromTemplate(_acl, _tokenManager, _tokenManager.MINT_ROLE());
+    }
+
+    function _mintTokens(ACL _acl, TokenManager _tokenManager, address[] _holders, uint256 _stake) internal {
+        _createPermissionForTemplate(_acl, _tokenManager, _tokenManager.MINT_ROLE());
+        for (uint256 i = 0; i < _holders.length; i++) {
+            _tokenManager.mint(_holders[i], _stake);
+        }
+        _removePermissionFromTemplate(_acl, _tokenManager, _tokenManager.MINT_ROLE());
+    }
+
+    function _mintTokens(ACL _acl, TokenManager _tokenManager, address _holder, uint256 _stake) internal {
```

Comment:
I don't know if we should bother with this version of the command. If we do, I would also rename it to just `_mintToken` since all it can ever mint is one token to one address.

Reply:
@sohkai 
1) I think that this function is handy because it encapsulates the complexity of having to grant permissions and then revoke them during the mint, so I would keep it.
2) Regarding the rename, the function only mints to one beneficiary address, yes, but it mints any amount of tokens, not just one, so you are actually minting "tokens" to that address.

Reply:
Good call on the name, let’s leave it as an overload 👍 .